server/vue-worker.js

```diff
@@ -47,22 +47,22 @@ async function render(context) {
 			}
 		});
 
-	const setCookies = [];
+	let setCookies = [];
 
 	return Promise.all([typesPromise, cookiePromise])
 		.then(([types, cookieInfo]) => {
 			// add fetched types to rendering context
 			context.config.graphqlPossibleTypes = types;
 			// update cookies in the rendering context with any newly fetched session cookies
 			context.cookies = Object.assign(context.cookies, cookieInfo.cookies);
-			// forward any newly fetched 'Set-Cookie' headers
-			context.setCookies = [...cookieInfo.setCookies];
+			// collect any newly fetched 'Set-Cookie' headers to send after the render
+			setCookies = [...cookieInfo.setCookies];
 			// render the app
 			return renderer.renderToString(context);
 		})
 		.then(html => {
 			// collect any cookies created during the app render
-			setCookies.concat(context.setCookies);
+			setCookies = [...setCookies, ...context.setCookies];
```

Comment:
will setCookies exist on the context if we're not setting them above? I'm getting a `uncaughtException: context.setCookies is not iterable` locally.

Reply:
I def see setCookies on the context and it's an array

Reply:
When did you see that? I haven't been able to reproduce that error. I can add a check to be safe though.

Reply:
I also see them set on the html response so It may be this only happened in the error case locally.

Reply:
I think the error case was triggered b/c I still had a browser window open when I started the dev server after updating the branch... so that page was calling with hmr constantly until something was available to respond...